lib/index.js

```diff
@@ -5,7 +5,63 @@ const spawn = require('child_process').spawn;
 const spawnSync = require('child_process').spawnSync;
 const JSON = require('./json-buffer');
 const isElectron = require('is-electron');
+const fs = require('fs');
+const os = require('os');
+const index = [0];
+const timeStamp = new Date().toISOString();
+const net = require('net');
+let clientConnected = false;
+let client;
+const READFILETIMELIMIT = 5000;
+const debugMode = false;
 
+const responseFolder = path.join(os.tmpdir(), 'sync_rpc_responses');
+if (!fs.existsSync(responseFolder)) {
+  fs.mkdirSync(responseFolder, {recursive: true});
+}
+
+const responseFile = path.join(responseFolder, `responseFile.txt`);
+
+function timeLimitExceeded(startTime) {
+  return startTime + READFILETIMELIMIT < Date.now();
+}
+
+function responseFileReadOnly() {
```

Comment:
ReadOnly or Readable?

Reply:
Explained in the description in line 20, it should be read only.